schema/verify/database_functions/create_portal_app_user.sql

```diff
@@ -2,6 +2,6 @@
 
 begin;
 
-select true from pg_roles where rolname='portal_app';
+select false from pg_roles where rolname='portal_app';
```

Comment:
```suggestion
select false from pg_roles where rolname='portal_app';
select true from pg_roles where rolname='ciip_portal'; 
```

Reply:
... although this might make sqitch ignore the result of the first statement now ...

Reply:
indeed it does